clients/mssql/dialect/dialect.go

```diff
@@ -56,6 +56,8 @@ func (MSSQLDialect) DataTypeForKind(kindDetails typing.KindDetails, isPk bool) s
 	case typing.ETime.Kind:
 		switch kindDetails.ExtendedTimeDetails.Type {
 		case ext.TimestampTzKindType:
+			return "datetimeoffset"
```

Comment:
It looks like before, if we received a timestamp with tz/offset info, we discarded the tz and stored it without it. Now we'll start storing the tz. Is this change going to result in ddl changes to the column type of existing columns?

Reply:
Yes it will. I'll migrate the tables that are in MSSQL from `datetime2` -> `datetimeoffset` once this change is in

Reply:
Actually, I don't think this would impact existing columns. 

If the existing column is created as `datetime2`, we will then tag the column as `TIMESTAMP_NTZ`.
When we merge our in-memory columns with existing, [existing will take precedence](https://github.com/artie-labs/transfer/blob/3c03abf1e0299e7aa37de0d0b4f01a6b51e804bd/lib/optimization/table_data.go#L297).

It is then on the SDK to discard the tz information, which it should be doing already.

Reply:
Ah okay so we won't start storing the tz info for existing columns, but if a new column is created that should be tz-aware then we will store it. Seems reasonable 👍